public/js/public/logout.js

```diff
@@ -12,9 +12,14 @@ $(document).ready(function () {
         },
         success: function (response) {
           localStorage.removeItem('token-login');
+          signOut();
           window.location.href = 'http://' + window.location.hostname;
         }
       });
     }
   });
 })
+function signOut() {
+  var auth2 = gapi.auth2.getAuthInstance();
```

Comment:
cái này ở đâu ra em?

Reply:
https://developers.google.com/identity/sign-in/web/sign-in, em làm theo cái link ni đây ạ